src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -469,8 +581,10 @@ def test_invalid_builtin_kernel():
 
 
 def test_lfricbuiltin_cma(dist_mem):
-    ''' Check that an LFRicBuiltIn returns None for CMA type (because
-    built-ins don't work with CMA operators). '''
+    '''Check that an 'LFRicBuiltIn' returns None for CMA type (because
```

Comment:
`'None'`?

Reply:
Well spotted. I think this is the only case in examples where quotes are not used.